data_collection/gazette/spiders/mg_belo_horizonte.py

```diff
@@ -0,0 +1,47 @@
+import datetime
+from urllib.parse import urlencode
+
+import scrapy
+from dateutil.rrule import DAILY, rrule
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgBeloHorizonteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3106200"
+
+    name = "mg_belo_horizonte"
+    allowed_domains = ["dom-web.pbh.gov.br"]
+    start_date = datetime.date(1995, 9, 26)
+    end_date = datetime.date.today()
+
```

Comment:
When testing a full crawl I get status code 429 (Too many requests) as soon as 1998. And then most requests fail :(

I was able to solve it with some `DOWNLOAD_DELAY`, though...
```suggestion
    end_date = datetime.date.today()
    custom_settings = {"DOWNLOAD_DELAY": 0.5}
```